jdaviz/configs/specviz2d/helper.py

```diff
@@ -195,7 +197,13 @@ def load_data(self, spectrum_2d=None, spectrum_1d=None, spectrum_1d_label=None,
                         color='warning', sender=self, timeout=10000)
                 self.app.hub.broadcast(msg)
 
+            else:
+                load_1d = True
+
         else:
+            load_1d = True
```

Comment:
Why do you set `load_1d = True` here  on the else condition for line 156, i.e. when `spectrum_2d` is provided? Everything works as expected.  Just trying to understand the flow here.

Reply:
This line only gets run when `spectrum_2d` is *not* provided (`None`). So I expect you will see the 1D loaded but nothing shown for the 2D viewer. Unless I have misunderstood the code?

Reply:
I meant the `load_1d` at line 204.  And yeah I think I see what you mean now.  It allows the use case when only the 1d is provided but not the 2d.  But would this cause an error at line 207, for the case when neither 2d nor 1d file is provided?  In that case, looks like `load_1d` gets set to true, the 1d spectrum tries to load but `spectrum_1d` is None?

Reply:
But why would someone try to `load_data` without providing the data?

Reply:
Good catch on the logic though. I added a check above. Does that look good to you?

Reply:
who knows **why** some people do the things they do.  I'm always of the opinion that someone somewhere will try the thing you think no one will try, and thus best safeguard against it.  Haha.   But this all looks good to me!